content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -351,7 +351,7 @@ different Kubernetes components.
 | `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
 | `VolumeScheduling` | `true` | Beta | 1.10 | 1.12 |
 | `VolumeScheduling` | `true` | GA | 1.13 | - |
-| `VolumeSubpath` | `true` | GA | 1.13 | - |
+| `VolumeSubpath` | `true` | GA | 1.10 | - |
```

Comment:
This one's weird because the feature has existed since 1.3, but we only added the feature gate for it as part of the CVE fix. It also means that the feature gate exists back in 1.7 patch releases.

If the intention of this page is to document when the actual feature gates were introduced, then I guess 1.10 works.

Reply:
Yes, it is a special case. 

To be more accurate.

- `feature` is since 1.3
- `feature-gate` is since 1.10 and backported to 1.7-1.9 for CVEs fix.